packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/index.scss

```diff
@@ -0,0 +1,365 @@
+.load-strategy {
+    &__wrapper {
+        position: fixed;
+        top: 40px;
+        z-index: 10;
+        width: 100%;
+
+        .dc-mobile-full-page-modal {
+            @include mobile {
+                width: 100%;
+            }
+
+            &__body {
+                height: 100%;
+            }
+        }
+
+        & .dc-tabs {
+            @include mobile {
+                height: 100%;
+                display: unset;
+                flex: 1;
+
+                &__content {
+                    height: calc(100% - 40px);
+                }
+            }
+        }
+    }
+
+    &__container {
+        @include desktop {
+            height: calc(80vh - 0.8rem - 4rem - 4.8rem - 5.8rem);
```

Comment:
could you pleas explain what are we calculating here?

Reply:
<img width="1156" alt="image" src="https://user-images.githubusercontent.com/97010868/196414248-25d6ec14-792c-4af1-b0ff-00a3b39229ab.png">
height of this strategy container this was a legacy change